lib/players/human.rb

```diff
@@ -0,0 +1,10 @@
+class Player::Human < Player
+
+  #gets player move, inherits from Player. 
+  def move(board)
+    puts "What's your next move?"
+    player_move = gets.chomp
+    player_move
```

Comment:
You don't need to store your return value in a local variable; just utilize the implicit return value of the expression.
```
def move(board)
  puts "What's your next move?"
  gets.chomp
end
```